website_payment_v10/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+==================================================
+Payment: Website Integration (Backported From v10)
+==================================================
+
+This module updates the functionality of ``website_payment`` to support
+changes introduced in ``website_portal_v10``. See README of
+``website_portal_v10`` about rationale for this module and some useful
+warnings.
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Uninstall ``website_portal``, if it was installed, and don't install it
+   again as long as this module is installed.
+
```

Comment:
Uninstall ``website_portal`` ? Are you sure ;-)

![2016-07-22_1-53-53](https://cloud.githubusercontent.com/assets/8736623/17042611/42f95438-4faf-11e6-9990-c917c9cee3ee.png)